### PR TITLE
iPad OS ctrl + c in terminal 

### LINKED ISSUE #114009: iPadOS: Ctrl + C in terminal doesn’t kill running process
## Description 

Hitting `ctrl` + `c` with the terminal focused and a process running (i.e. `yarn start` in a React app) does not kill the process. 

If someone is able to provide a good starting point, I would be willing to take a stab at this.

- VSCode Version: 1.53.0-insider
- OS Version: iPadOS 14.1 (18A8395)
- Commit: 01089c0a5052432594a177f90ad9f3148ec2a6a2
- Date: 2021-01-07T05:25:51.740Z (16 hrs ago)
- Browser: Mozilla/5.0 (iPad; CPU OS 14_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) CriOS/87.0.4280.77 Mobile/15E148 Safari/604.1

## Steps to Reproduce:

1. On iPad, open Codespaces environment in Chrome
2. Open integrated Terminal
3. Start application (i.e. like a create-react-app - `yarn start`)
4. Focus terminal and hit `ctrl` + `c`

## Expected

Process (`yarn start`) should stop running

## Actual
Terminal receives new line

https://user-images.githubusercontent.com/3806031/103948200-a11ad800-50f6-11eb-8cb4-87e951735911.MP4

Does this issue occur when all extensions are disabled?: Yes

Related:
- https://github.com/cdr/code-server/issues/1343

### PR DESCRIPTION
**Pull request by meganrogge**
This PR fixes #114009

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "code-oss-dev",
-  "version": "1.55.0",
```
Comment (rebornix) [inline package.json:3]: we want to remove this change.

Comment (meganrogge) [reply package.json:3]: good catch thanks - happened when I changed the base branch to release/1.55

File: src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts [hunk 2]
```diff
@@ -125,7 +125,7 @@ if (platform.isWindows) {
 if (isIPad) {
 	registerSendSequenceKeybinding(String.fromCharCode('C'.charCodeAt(0) - CTRL_LETTER_OFFSET), { // ctrl+c
 		when: ContextKeyExpr.and(KEYBINDING_CONTEXT_TERMINAL_FOCUS),
-		primary: KeyMod.CtrlCmd | KeyCode.KEY_C
+		primary: KeyCode.Ctrl | KeyCode.KEY_C
```
Comment (rebornix) [inline src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts:128]: Is there any particular reason that we use `KeyCode.Ctrl` here but not `KeyMod.WinCtrl`?

Comment (meganrogge) [reply src/vs/workbench/contrib/terminal/browser/terminal.contribution.ts:128]: you're right, I think it should be winCtrl, fixing

**Review by rebornix — APPROVED** [review]

**Review by Tyriar — APPROVED** [review]

**Pull request merged**